### DISCUSSION TITLE
How to view Revit 2d drawings/sheets in itwin viewer?

### DISCUSSION THREAD

**Discussion by tpecnoc** [iTwin/itwinjs-core#4575] [Q&A]
I am a newbie on Itwin platform. I exported a Revit file to iModelhub using Itwin synchronizer. Is it possible to view the revit 2D drawings/sheets in Itwin viewer besides the 3D? If yes, how? Thanks.

**Comment by aruniverse** [comment]
Hi! 

Yes you should be able to view your sheets and drawings within the iTwin Viewer. I recommend you take a look at this [2D Sample](https://www.itwinjs.org/sandboxes/iTwinPlatform/2d%20Viewer)
